chainer/function_node.py

```diff
@@ -593,3 +595,233 @@ def delete_hook(self, name):
             del self.local_function_hooks[name]
         else:
             raise KeyError('Hook %s does not exist' % name)
+
+
+def grad(outputs, inputs, grad_outputs=None, grad_inputs=None, set_grad=False,
+         retain_grad=False, enable_double_backprop=False):
+    """Computes the gradient of output variables w.r.t. the input variables.
+
+    This function implements the backpropagation algorithm. While
+    :meth:`Variable.backward` also implements backprop, this function selects
+    the smallest paths in the computational graph needed to compute the
+    gradients w.r.t. inputs. The error is backpropagated only through these
```

Comment:
Could you replace `w.r.t. ` with `w.r.t.\\ ` (please note that one blank space is needed after double backslash `\\`)?